cmd/manager/operator.go

```diff
@@ -326,6 +330,53 @@ func ensureDefaultScanSettings(ctx context.Context, crclient client.Client, name
 	return lastErr
 }
 
+// createServiceMonitor attempts to create a ServiceMonitor out of service, and updates it to include the controller
+// metrics paths.
+func createServiceMonitor(ctx context.Context, cfg *rest.Config, mClient *monclientv1.MonitoringV1Client,
+	namespace string, service *v1.Service) error {
+	ok, err := k8sutil.ResourceExists(discovery.NewDiscoveryClientForConfigOrDie(cfg),
+		"monitoring.coreos.com/v1", "ServiceMonitor")
+	if err != nil {
+		return err
+	}
+	if !ok {
+		log.Info("Install prometheus-operator in your cluster to create ServiceMonitor objects")
+		return nil
+	}
+
+	serviceMonitor := metrics.GenerateServiceMonitor(service)
+	for i, _ := range serviceMonitor.Spec.Endpoints {
```

Comment:
You can just omit the second loop variable instead of ignoring it with the blank assignment.

Reply:
fixed